storageapi/src/vespa/storageapi/messageapi/storagemessage.cpp

```diff
@@ -160,39 +151,50 @@ createAddress(vespalib::stringref cluster, const lib::NodeType& type, uint16_t i
     return os.str();
 }
 
+size_t
+calculate_node_hash(const lib::NodeType& type, uint16_t index)
+{
+    struct NodeValue {
+        uint16_t type;
+        uint16_t index;
+        NodeValue(const lib::NodeType& type_in, uint16_t index_in) : type(type_in), index(index_in) {}
+    };
+    NodeValue value(type, index);
+    return vespalib::hashValue(&value, sizeof(NodeValue));
```

Comment:
To avoid having to contemplate any struct alignment issues and how they may affect the resulting value, could be simplified down to
```c++
uint16_t buf[] = { type, index };
return vespalib::hashValue(&buf, sizeof(buf));
```

Reply:
Nice, will fix.